package io.github.rohrl.interstellar.mixin.client;

import com.mojang.blaze3d.systems.RenderSystem;
import org.joml.Vector3f;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(value=RenderSystem.class,remap=false)
public interface ShaderLightsAccessor {
    @Accessor("shaderLightDirections") static Vector3f[] interstellar$lights() {throw new AssertionError();}
}
